src/igvfd/searches/configs/tabular_file.py

```diff
@@ -31,6 +31,15 @@ def tabular_file():
             'assay_titles': {
                 'title': 'Assay'
             },
+            'integrated_in.file_set_type': {
+                'title': 'Library Type'
+            },
+            'integrated_in.associated_phenotypes.term_name': {
+                'title': 'Associated Phenotype'
+            },
+            'integrated_in.small_Scale_gene_list.symbol': {
```

Comment:
typo in small_scale_gene_list, second s should lowercase

Reply:
fixed